utilities/src/main/java/io/syndesis/qe/resource/Resource.java

```diff
@@ -0,0 +1,6 @@
+package io.syndesis.qe.resource;
+
+public interface Resource {
```

Comment:
This is great, but would it also make sense to add a `waitUntilReady` method? Or perhaps `isReady` and then keep the waits in a separate class, but with just a call to `isReady`?

Reply:
looks good, I'll check that out